src/lender/adapters/mkr/assessor.sol

```diff
@@ -0,0 +1,90 @@
+// Copyright (C) 2020 Centrifuge
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity >=0.5.15 <0.6.0;
+
+import "./../../assessor.sol";
+
+interface ClerkLike {
+    function remainingCredit() external view returns (uint);
+    function juniorStake() external view returns (uint);
+    function remainingCreditCollateral() external view returns (uint);
+}
+
+contract MKRAssessor is Assessor {
+    ClerkLike public clerk;
+
+    function depend(bytes32 contractName, address addr) public auth {
+        if (contractName == "clerk") {
+            clerk = ClerkLike(addr);
+        } else {
+            super.depend(contractName, addr);
+        }
+    }
+
+    // calculates the tokenPrice based on the nav and the reserve
+    function calcSeniorTokenPrice(uint nav_, uint) public view returns(uint) {
```

Comment:
what is the name of the sec param?

Reply:
We are not using it in the wrapper otherwise it would be a warning.